protocol/migrations/sqlite/1714913596_remove_status_community.up.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM communities_communities WHERE id = X'039b2da47552aa117a96ea8f1d4d108ba66637c7517a3c94a57b99dbb8a002eda2';
+DELETE FROM communities_events WHERE id = X'039b2da47552aa117a96ea8f1d4d108ba66637c7517a3c94a57b99dbb8a002eda2';
+DELETE FROM communities_shards WHERE community_id = X'039b2da47552aa117a96ea8f1d4d108ba66637c7517a3c94a57b99dbb8a002eda2';
```

Comment:
@Parveshdhull I'm not sure about the validity of my feedback, but to be correct, shouldn't the migration also delete records in the following tables? They all contain a column named `community_id`.

- `activity_center_notifications`
- `applied_community_events`
- `collapsed_community_categories`
- `communities_check_channel_permission_responses`
- `communities_control_node`
- `communities_permission_token_criteria_results`
- `communities_requests_to_leave`
- `community_grants`
- `community_tokens`
- `curated_communities`
- `encrypted_community_description_cache`
- `encrypted_community_description_id_and_clock`
- `encrypted_community_description_missing_keys`
- `profile_showcase_communities_contacts`
- `profile_showcase_communities_preferences`

Reply:
Hi @ilmotta, Thank you very much for your valuable feedback and sharing these tables.
Probably we should also delete from these table too, but I am not sure as DeleteCommunity function only deletes from above three :thinking: 

https://github.com/status-im/status-go/blob/9c2c638aaf91afa3b68b0572d57499746a6321a7/protocol/communities/persistence.go#L219

Maybe @cammellos will have more info.

Reply:
> Maybe @cammellos will have more info.

Confirmed. Removing from these 3 tables should be enough.

Reply:
Great! Thanks for taking the time to check @Parveshdhull